tests/integ/scala/test_file_operation_suite.py

```diff
@@ -219,6 +222,55 @@ def test_put_negative(session, temp_stage, temp_source_directory, path1):
     assert "does not exist or not authorized." in str(stage_not_exist_info)
 
 
+def test_put_stream_with_one_file(session, temp_stage, path1, path2, path3):
```

Comment:
Let's skip these tests for stored proc. One example could be found in https://github.com/snowflakedb/snowpark-python/blob/c838f17c3fb373f7f3cf2192c6a5bf2d74a0dc19/tests/integ/test_session.py#L36-L39